packages/lisk-p2p/src/p2p_types.ts

```diff
@@ -13,28 +13,58 @@
  *
  */
 /* tslint:disable:no-empty-interface*/
-export interface P2PMessagePacket {}
 
-export interface P2PRequestPacket {}
+export interface P2PRequestPacket<T> {
+	readonly procedure: string;
+	readonly params?: T;
+}
+
+export interface P2PResponsePacket {
+	readonly data: unknown;
+}
 
-export interface P2PResponsePacket {}
+export interface P2PMessagePacket<T> {
+	readonly event: string;
+	readonly data: T;
+}
+
+export interface PeerInfo {
+	readonly ipAddress: string;
+	readonly wsPort: number;
+}
+
+export interface NodeInfo {
```

Comment:
I renamed to `P2PNodeStatus`; it can be the same type as the argument passed to the `P2P` module's `setNodeStatus(nodeStatus: P2PNodeStatus): void` method.